doc/cephfs/scrub.rst

```diff
@@ -131,3 +131,15 @@ Control (ongoing) File System Scrubs
    {
        "return_code": 0
    }
+
+Evaluate strays using recursive scrub
+=====================================
+
+- In order to evaluate strays i.e. purge stray directories in ``~mdsdir`` use the following command::
+
```

Comment:
Is `~mdsdir` a specific CephFS thing ? With the tilde it looks like a user homedir

Reply:
@anthonyeleven yes, `~mdsdir` contains stray dirs which is kind of CephFS exclusive

Reply:
Groovy, thanks.